pylint/checkers/refactoring.py

```diff
@@ -0,0 +1,77 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+"""check to consider a refactory of code"""
+
+import collections
+
+import astroid
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages, is_builtin_object, safe_infer
+
+
+MSGS = {
+    'R1701': (
+        "isinstances of '%s' are not merged. "
+        "Consider merging 'isinstance(%s, (class1, class2, ...))'",
+        "consider-merging-isinstance",
+        "Used when isinstance methods are not grouped by object"),
+}
+
+
+def get_duplicated(items):
```

Comment:
If this is private, you can drop the documentation. The name is enough for determining what it does.